### PR TITLE
chore(LogViewer): updated imports and docs for extracted log viewer

### LINKED ISSUE #3185: Extract Log Viewer and update org extension docs accordingly
- extract the Log Viewer package from the PF React repo
- restructure Log Viewer docs with patternflly-docs
- update org accordingly to keep Log Viewer docs being displayed properly

### PR DESCRIPTION
**Pull request by wise-king-sullyman**
Closes #3185

### CODE REVIEW

File: packages/v4/package.json [hunk 1]
```diff
@@ -21,7 +21,8 @@
     "@patternfly/quickstarts": "^2.2.4",
     "@patternfly/react-docs": "5.96.8",
     "react": "^16.8.0 || ^17.0.0",
-    "react-dom": "^16.8.0 || ^17.0.0"
+    "react-dom": "^16.8.0 || ^17.0.0",
+    "@ausuliv/react-log-viewer": "^4.84.2"
```
Comment (jschuler) [inline packages/v4/package.json:25]: what changes are in your forked version? Shouldn't bring in forked versions if possible

File: packages/v4/patternfly-docs/patternfly-docs.source.js [hunk 2]
```diff
@@ -38,16 +38,19 @@ module.exports = (sourceMD, sourceProps) => {
...
       .resolve('@patternfly/react-charts/package.json')
       .replace('package.json', 'src');
+
   const reactLogViewerPath = require
-      .resolve('@patternfly/react-log-viewer/package.json')
-      .replace('package.json', 'src');
+      .resolve('@ausuliv/react-log-viewer/package.json')
+  const logViewerContentBase = path.join(reactLogViewerPath.replace('package.json', 'patternfly-docs'), '/content/extensions/react-log-viewer')
+  const logViewerPropsBase = reactLogViewerPath.replace('package.json', 'src')
+  const logViewerPropsIgnore = ['**/*.test.tsx', '**/examples/*.tsx'];
```
Comment (DaoDaoNoCode) [inline packages/v4/patternfly-docs/patternfly-docs.source.js:46]: Why do we need `logViewerPropsIgnore` here? It's totally the same as `reactPropsIgnore`.

File: packages/v4/package.json (hunk 1 above)
Comment (wise-king-sullyman) [reply packages/v4/package.json:25]: No changes, the forked version is the extracted log viewer pulled out of the pf-react repo into its own repo as that was the main goal here.

File: packages/v4/patternfly-docs/patternfly-docs.source.js (hunk 2 above)
Comment (wise-king-sullyman) [reply packages/v4/patternfly-docs/patternfly-docs.source.js:46]: Ah yes, looks like this probably isn't needed to me as well. I had copied this element of the approach from Cooper's quickstarts docs PR.

**Comment by wise-king-sullyman** [comment]
Closing as the docs were added in #3365

**Pull request closed without merging**